test/xpu_api/numerics/numeric.ops/reduce/xpu_reduce_init_op.pass.cpp

```diff
@@ -0,0 +1,90 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Will it be better if we combine three files into one? Those are pretty similar. Most of code in these files is common

Reply:
Agree. I will combine them.